stdlib/gc.ml

```diff
@@ -67,31 +67,29 @@ external huge_fallback_count : unit -> int = "caml_gc_huge_fallback_count"
 external eventlog_pause : unit -> unit = "caml_eventlog_pause"
 external eventlog_resume : unit -> unit = "caml_eventlog_resume"
 
-open Printf
-
 let print_stat c =
   let st = stat () in
```

Comment:
This one is possibly nicer with an auxiliary similar to the one you temporarily introduced in the commit history:

```suggestion
  let st = stat () in
  let print fmt = Printf.fprintf c (fmt ^^ "\n") in
```

Reply:
I've applied your suggestion